docs-src/go/introduction-to-go-sdk.md

```diff
@@ -46,6 +42,7 @@ We recommend that developers possess at least a moderate level of experience in
 - Goroutines
 - Channels
 - Concurrency patterns - parallelism, producer-consumer, etc.
+  - The Temporal Go SDK provides [deterministic wrappers](https://community.temporal.io/t/workflow-determinism/4027) for Go's concurrency primitives
```

Comment:
```suggestion
  - The Temporal Go SDK provides [deterministic wrappers](https://community.temporal.io/t/workflow-determinism/4027) for Go's concurrency primitives
```

Is there a place in the docs we can link to instead? I am not a huge fan of linking to community support threads if we can avoid it.